modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/main_view/modal/share_modal_content/ShareModalContent.tsx

```diff
@@ -136,14 +145,7 @@ function CollaboratorListItem({
 				)}
 			</div>
 
-			<div className="autofit-col">
-				<PermissionSelector
-					actionIds={actionIds}
-					onChange={handleChangeUserProperties}
-				/>
-			</div>
-
-			<div className="autofit-col">
+			<div className="autofit-col dropdown-options">
```

Comment:
add class `p-0` and remove css related to this 🙏

Reply:
Thanks for the tip! I think I tried `c-p-0` and it would get overwritten, but it looks like `p-0` works 👍